src/script/view_model/panel/NotificationsPanel.tsx

```diff
@@ -44,44 +46,69 @@ const NotificationsPanel: React.FC<NotificationsPanelProps> = ({
   conversationState = container.resolve(ConversationState),
 }) => {
   const {activeConversation} = useKoSubscribableChildren(conversationState, ['activeConversation']);
-  const {notificationState} = useKoSubscribableChildren(activeConversation, ['notificationState']);
-
+  const {notificationState} = useKoSubscribableChildren(activeConversation!, ['notificationState']);
+  const saveOptionNotificationPreference = (value: number) => {
+    repositories.conversation.setNotificationState(activeConversation!, value);
+  };
   const [scrollbarRef, setScrollbarRef] = useEffectRef<HTMLDivElement>();
   useFadingScrollbar(scrollbarRef);
 
   const [settings] = useState(
-    Object.values(NOTIFICATION_STATE).map(status => ({
-      text: getNotificationText(status),
-      value: status,
-    })),
+    Object.values(NOTIFICATION_STATE).map(status => {
+      return {
+        label: getNotificationText(status),
+        value: status,
+      };
+    }),
```

Comment:
So let's do smth like that:

```suggestion
    Object.values(NOTIFICATION_STATE).map(status => ({
        label: getNotificationText(status),
        value: status,
      })),
```